Presentations/06-R-programming-practices.Rmd

```diff
@@ -769,9 +688,11 @@ name: looping
 
 # Looping
 
+## Loops
+
 * One thing that usually gives people away as Stata users writing R code are loops
 
-* In Stata, we use `for` loops quite a lot
+* In Stata, we use `foreach` loops quite a lot
```

Comment:
`for` would be ok, since it could either be foreach or forvalues in state.

Reply:
solved in 7d50441